dev/io.openliberty.microprofile.lra.1.0.internal_fat_tck/publish/servers/LRATCKServer/server.xml

```diff
@@ -0,0 +1,27 @@
+<!--
+    Copyright (c) 2020 IBM Corporation and others.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+   
+    Contributors:
+        IBM Corporation - initial API and implementation
+ -->
+<server>
+   <featureManager>
+        <feature>mpLRA-1.0</feature>
+        <feature>cdi-2.0</feature>
+        <feature>jaxrs-2.1</feature>
+        <feature>mpConfig-1.3</feature>
```

Comment:
is MP Config 1.3 correct? Shouldn't it be at least 1.4?

Reply:
Actually? Is MP Config needed at all?

Reply:
yes mpconfig is needed, and the pom for the tck artifacts on maven specifies 1.3, so it is probably correct to use it here